mlflow/tracking/client.py

```diff
@@ -52,18 +52,35 @@ def create_run(self, experiment_id, user_id=None, run_name=None, source_type=Non
                      :py:class:`mlflow.entities.RunTag` objects.
         :return: :py:class:`mlflow.entities.Run` that was created.
         """
+
         tags = tags if tags else {}
+
+        # Extract run attributes from tags
+        # This logic is temporary; by the 1.0 release, this information will only be stored in tags
+        # and will not be available as attributes of the run
+        parent_run_id = tags.get(MLFLOW_PARENT_RUN_ID)
```

Comment:
`parent_run_id` we actually intentionally added as an extra parameter to the create_run call. The idea here was that the notion of linking two runs may be fundamental to the creation of those runs, and may imply other properties (for example, we might update metadata on the parent run).

In practice, we haven't yet implemented any such extra linking from an API perspective, so it may be OK to degrade this to just be a tag instead of a property of the create call.

Reply:
The parent run ID does seem like something that we might want to have as a strongly integrated attribute on runs, but while it is exposed as a 'top level' attribute when creating runs, it seems to only be accessible as a tag when getting a run.

I would suggest that it should either be 'just' a tag and set as in this PR, or it should be promoted to a full attribute like `start_time` that appears in the `RunInfo` object itself.

Since it's currently stored as a tag, and adding it to the `RunInfo` object will require changing the protobuf files etc., I suggest keeping it as a tag for the time being and promoting it to be an actual attribute later if we think it's valuable.

Reply:
(FYI my last commit is not meant to assume the above decision - just don't want to forget to update the docs in the case my proposal is accepted)

Reply:
Alright, sounds reasonable to me.